CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## CHANGELOG
 
+### 0.4.0
+
+* updating to RC2 standard of JSON API
```

Comment:
Not sure if it matters, but 4.0 was RC2, 4.1 was RC2.1? where linked was changed to included and resource became related. 4.1 is somewhat backwards compatible in that you could override what those properties are called, but 4.2 is where it became RC3, which includes the consistent linkage format. If you don't care about the history, I can just call the current version 4.0 and update the changelog accordingly.